litmus-portal/frontend/src/components/PredifinedWorkflows/data.ts

```diff
@@ -20,13 +20,58 @@ export default [
     urlToIcon:
       'https://hub.litmuschaos.io/api/icon/1.7.0/generic/node-memory-hog.png',
     chaosWkfCRDLink:
-      'https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/workflows/node-memory-hog/workflow.yaml',
+      'https://raw.githubusercontent.com/ishangupta-ds/chaos-workflows/master/Argo/argowf-native-node-memory-hog.yaml',
 
     gitLink:
-      'https://github.com/litmuschaos/chaos-charts/blob/master/workflows/node-memory-hog/workflow.yaml',
+      'https://github.com/ishangupta-ds/chaos-workflows/blob/master/Argo/argowf-native-node-memory-hog.yaml',
     provider: 'MayaData',
     description: 'Injects a memory spike on a node',
-    totalRuns: 5300,
+    totalRuns: 4300,
+    isCustom: false,
+  },
+  {
+    workflowID: '3',
+    title: 'pod-cpu-hog',
+    urlToIcon:
+      'https://hub.litmuschaos.io/api/icon/1.7.0/generic/pod-cpu-hog.png',
+    chaosWkfCRDLink:
+      'https://raw.githubusercontent.com/ishangupta-ds/chaos-workflows/master/Argo/argowf-native-pod-cpu-hog.yaml',
```

Comment:
Change all links to litmus repo

Reply:
It is not possible to do that as we don't have these in litmuschaos org. @ksatchit will be changing 2 of the templates then we will raise a PR for the same. We can change it after that. @gdsoumya